src/components/utrecht/TableBody.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2022 Robbert Broersma
+ */
+
+import clsx from "clsx";
+import { ForwardedRef, forwardRef, PropsWithChildren, TableHTMLAttributes } from "react";
+
+type TableBodyProps = TableHTMLAttributes<HTMLTableSectionElement>;
+
+export const TableBody = forwardRef(
+  (
+    { children, className, ...restProps }: PropsWithChildren<TableBodyProps>,
+    ref: ForwardedRef<HTMLTableSectionElement>
+  ) => (
+    <tbody {...restProps} ref={ref} className={clsx("utrecht-table__body", className)}>
+      {children}
+    </tbody>
+  )
+);
+
+TableBody.displayName = "utrecht-table__body";
```

Comment:
Can we kebab-case displaynames instead of bem-casing them? It will be the name of the webcomponent and then it sorta matches the component name with an organisation prefix?